make/autoconf/util.m4

```diff
@@ -239,12 +239,17 @@ AC_DEFUN([UTIL_GET_EPOCH_TIMESTAMP],
     timestamp=$($DATE --utc --date=$2 +"%s" 2> /dev/null)
   else
     # BSD date
-    timestamp=$($DATE -u -j -f "%FZ %TZ" "$2" "+%s" 2> /dev/null)
+    # ISO-8601 date&time in Zulu 'date'T'time'Z
+    timestamp=$($DATE -u -j -f "%FT%TZ" "$2" "+%s" 2> /dev/null)
```

Comment:
You are removing the space between FT and TZ, I'm just curious why and if that is significant.

Reply:
yes, this is supposed to be parsing an ISO8601 eg.2022-02-09T14:47:36Z, the previous JDK-8282769 had an error in this string